core/trino-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -1336,6 +1336,14 @@ export class QueryDetail extends React.Component {
                                             {query.queryStats.totalCpuTime}
                                         </td>
                                     </tr>
+                                    <tr>
+                                        <td className="info-title">
+                                            CPU Time (failed tasks)
+                                        </td>
+                                        <td className="info-text">
+                                            {query.queryStats.failedCpuTime}
+                                        </td>
+                                    </tr>
                                     <tr>
                                         <td className="info-title">
                                             Scheduled Time
```

Comment:
nit: Scheduled time is also something we might want to have for failed tasks. I wonder if we should add it.

Reply:
Makes sense. It is an omission - will add.